stubs/pyinstaller/PyInstaller/__init__.pyi

```diff
@@ -1,11 +1,11 @@
-from typing_extensions import LiteralString
+from typing_extensions import Final, LiteralString
 
 from PyInstaller import compat as compat
 
 __all__ = ("HOMEPATH", "PLATFORM", "__version__", "DEFAULT_DISTPATH", "DEFAULT_SPECPATH", "DEFAULT_WORKPATH")
-__version__: str
-HOMEPATH: str
-DEFAULT_SPECPATH: str
-DEFAULT_DISTPATH: str
-DEFAULT_WORKPATH: str
-PLATFORM: LiteralString
+__version__: Final[str]
```

Comment:
Given the special meaning of `__version__`, I feel like it should always be considered constant/final. Maybe a project-wide search&replace for `^__version__: str$` --> `__version__: Final[str]` could have value?

Reply:
Yes, sounds good